src/models/Project/createProject/index.js

```diff
@@ -35,7 +35,7 @@ const createProject = async ({
     errors.push({ field: 'name', message: nameValidation.error })
   }
 
-  const slug = slugify(trimmedName)
+  const slug = slugify(trimmedName, { lower: true })
```

Comment:
That was causing tests to fail. I removed it thinking it was default for the new slug library but it wasn’t.